frontend/src/app/testQueue/TestCardForm/TestCardForm.utils.tsx

```diff
@@ -227,43 +227,44 @@ export const convertFromMultiplexResponse = (
   });
 };
 
+const REQUIRED_AOE_QUESTIONS_BY_DISEASE: {
+  [key in AOEFormOption]: Array<keyof AoeQuestionResponses>;
```

Comment:
opted here to require an entry for every AOE option rather than doing something like `[key in AOEFormOption]?` that would allow for partial entries so that when we add a new form option, the dev adding will be forced to add things here in order for the frontend to compile